projects/common-form-elements/package.json

```diff
@@ -1,15 +1,21 @@
 {
-  "name": "common-form-elements-v9",
-  "version": "4.3.1",
-  "peerDependencies": {
-    "@angular/common": "^9.1.13",
-    "@angular/core": "^9.1.13",
-    "@angular/forms": "^9.1.13",
-    "@project-sunbird/client-services": "3.6.x",
-    "lodash-es": "^4.17.15"
-  },
-  "dependencies": {
-    "immutable": "^4.0.0-rc.12",
-    "moment": "^2.29.1"
-  }
-}
+    "name": "common-form-elements-v9",
+    "version": "4.3.1",
+    "peerDependencies": {
+        "@angular/common": "^9.1.13",
+        "@angular/core": "^9.1.13",
+        "@angular/forms": "^9.1.13",
+        "@project-sunbird/client-services": "3.6.x",
+        "lodash-es": "^4.17.15"
+    },
+    "peerDependenciesMeta": {
+        "@project-sunbird/ckeditor-build-classic": {
+            "optional": true
+        }
+    },
+    "dependencies": {
+        "immutable": "^4.0.0-rc.12",
+        "moment": "^2.29.1",
+        "@project-sunbird/ckeditor-build-classic": "4.1.3"
```

Comment:
@vaibhavbhuva remove `ckeditor-build-classic` from `dependencies`

Reply:
@pallakartheekreddy Done. Please check the same